dm/worker/subtask.go

```diff
@@ -650,3 +635,21 @@ func (st *SubTask) fail(err error) {
 		},
 	})
 }
+
+// HandleError handle error for syncer unit
+func (st *SubTask) HandleError(ctx context.Context, req *pb.HandleWorkerErrorRequest) (string, error) {
+	syncUnit, ok := st.currUnit.(*syncer.Syncer)
+	if !ok {
+		return "", terror.ErrWorkerOperSyncUnitOnly.Generate(st.currUnit.Type())
+	}
+
+	msg, err := syncUnit.HandleError(ctx, req)
+	if err != nil {
+		return msg, err
+	}
+
+	if st.Stage() == pb.Stage_Paused {
+		err = st.Resume()
```

Comment:
will there be more errors so failed the resume?

Reply:
Can you describe it in more detail? I can't understand.